src/proxy/query.go

```diff
@@ -110,13 +110,13 @@ func (spanner *Spanner) ComQuery(session *driver.Session, query string, bindVari
 		if qr, err = spanner.handleUseDB(session, query, node); err != nil {
 			log.Error("proxy.usedb[%s].from.session[%v].error:%+v", query, session.ID(), err)
 		}
-		spanner.auditLog(session, R, xbase.USEDB, query, qr)
+		spanner.auditLog(session, R, xbase.USEDB, query, qr, err != nil)
```

Comment:
Maybe we can define a flag which default value is` true` instead of 'err != nil', for example, we can use a `statusFlag` or `statusOK`, the code would be next:
```
        if qr, err = spanner.handleUpdate(session, query, node); err != nil {
            log.Error("proxy.update[%s].from.session[%v].error:%+v", xbase.TruncateQuery(query, 256), session.ID(), err)
            statusOK = false 
        }   
        spanner.auditLog(session, W, xbase.UPDATE, query, qr, statusOK)
```

Reply:
good advice